### PR TITLE
Change file structure to use src/

### PR DESCRIPTION
**Pull request by sambattalio**
# Description

The root directory was very messy so I started a file restructure to utilize a src/ directory.

In the files I append 'src' to the system path so you don't have to src. every time you import a file.

# Checklist:

- [X] My code follows the style of this project
- [X] I have performed a self-review of my own code
- [X] My code is self documenting
- [X] My changes generate no new major crashes/bugs
- [X] Any new features/functionality have corresponding test cases added

### CODE REVIEW

File: tests/test_autocomplete.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # test_autocomplete.py
 
 from customUrwidClasses import CommandBar
-from commandChanVim import urwidView
+from ..commandChanVim import urwidView
```
Comment (ginglis13) [inline tests/test_autocomplete.py]: I think this should be just commandChanVim bc tests are run from the top level directory. See output of travis

File: tests/test_init.py [hunk 2]
```diff
@@ -7,8 +7,10 @@
 
 from enum import Enum
 
+sys.path.append('src')
+
 from config import Config
-from viewClass import View
+from Views.viewClass import View
```
Comment (ginglis13) [inline tests/test_init.py]: I think this should be `from src.Views.viewClass import View`

File: tests/test_init.py [hunk 3]
```diff
@@ -18,7 +20,7 @@
 from commandHandlerClass import CommandHandler
 from customeTypes import LEVEL, MODE, SITE, STICKIES
 
-from commandChanVim import urwidView
+from ..commandChanVim import urwidView
```
Comment (ginglis13) [inline tests/test_init.py]: Same as above, tests are run from top level dir

**Comment by ginglis13** [comment]
Good stuff Sam thanks for organizing this

**Comment by sambattalio** [comment]
No problem. Thanks for the code review!

**Comment by sambattalio** [comment]
Looking forward to see @wtheisen's code review!

**Comment by wtheisen** [comment]
at the golf course rn I’ll review it in a couple hours

**Comment by wtheisen** [comment]
This definitely needed to be done at some point, thanks for taking it on your shoulders!

**Comment by sambattalio** [comment]
Thank you for your support and successful merge!

**Comment by ginglis13** [comment]
🚀

**Pull request merged**